docs/reference/migration/migrate_7_0/analysis.asciidoc

```diff
@@ -0,0 +1,6 @@
+[[breaking_70_analysis_changes]]
+=== Analysis changes
+
+==== The `delimited_payload_filter` is deprecated
+
+The `delimited_payload_filter` is deprecated and should be replaced by `delimited_payload`.
```

Comment:
Maybe we should make it clearer that the filter is just renamed, and its only the old name that is deprecated, something like "The `delimited_payload_filter` is renamed to `delimited_payload`, the old name is deprecated and will be removed at some point, so it should be replaces by `delimited_payload`" or slightly different.